tractolearn/config/Experiment.py

```diff
@@ -0,0 +1,197 @@
+# -*- coding: utf-8 -*-
```

Comment:
Use lower cases for any Python files:
https://peps.python.org/pep-0008/

Reply:
Done